src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,54 @@
-export const TodoFilter = () => (
+import React, { Dispatch, SetStateAction } from 'react';
+import { FilterType } from '../../types/Todo';
+
+type Props = {
+  filterBy: string,
```

Comment:
This should be limited only to possible cases, think of `const enum` type or `JSON` object holding all possible values, and use it here.

```
const enum Filter {
    ALL = 'All',
    ACTIVE = 'Active',
    COMPLETED = 'Completed'
}
```